ui/apps/dashboard/src/app/(organization-active)/(dashboard)/billing/(overview)/page.tsx

```diff
@@ -66,21 +84,37 @@ export default async function Page() {
           </div>
           {entitlementUsage.runCount.limit !== null && <LimitBar data={runs} className="my-4" />}
           <LimitBar data={steps} className="mb-6" />
-          {!overageAllowed && (
-            <Alert
-              severity="info"
-              className="mb-6 flex items-center justify-between text-sm"
-              link={
-                <NewButton
-                  appearance="outlined"
-                  kind="secondary"
-                  label="Upgrade plan"
-                  href={pathCreator.billing({ tab: 'plans', ref: 'app-billing-page-overview' })}
-                />
-              }
-            >
-              For usage beyond the limits of this plan, upgrade to a new plan.
-            </Alert>
+          <div className="border-subtle mb-6 border" />
+          <AddOn
+            title="Event Size"
+            value={
+              entitlementUsage.eventSize.limit >= 1024
+                ? `${(entitlementUsage.eventSize.limit / 1024).toFixed(2)} MB`
+                : `${entitlementUsage.eventSize.limit} KB`
+            }
+            canIncreaseLimitInCurrentPlan={entitlementUsage.isCustomPlan}
+            tooltipContent="The maximum size for a single event"
+          />
+          <AddOn
+            title="Concurrency"
+            value={entitlementUsage.concurrency.limit}
+            canIncreaseLimitInCurrentPlan={entitlementUsage.isCustomPlan}
+            tooltipContent="Maximum concurrently executing steps"
+          />
+          <AddOn
+            title="Log history"
+            value={`${entitlementUsage.history.limit} day${
+              entitlementUsage.history.limit === 1 ? '' : 's'
+            }`}
+            canIncreaseLimitInCurrentPlan={entitlementUsage.isCustomPlan}
+            tooltipContent="Tracing for every function run"
+          />
+          {entitlementUsage.isCustomPlan && (
```

Comment:
Ideally, we'd love to show this entitlement for all plans w/ a "contact us" button. This is an enterprise option and we want to be able to display it for all users if they're evaluating from Free to Enterprise (all plans)